src/transactional-mutations-plugin/transactional-mutations-messages.ts

```diff
@@ -0,0 +1,6 @@
+export const transactionalMutationsMessages = {
```

Comment:
Change the consts so every message will be a const on their own, rather than be a field inside a const object.
Because of that, change their names to be like `PLUGIN_STARTED_JOB`

Reply:
i just copied it from th elogger-plugin-messages...

Reply:
so decide how it should be, because in the logger plugin messages it dosent like this

Reply:
@DoctorVoid @chenshoo what do you think?